index.html

```diff
@@ -5,14 +5,80 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
     <meta http-equiv="X-UA-Compatible" content="ie=edge" />
     <title>Spotify Clone</title>
-    <!-- don't forget to link your styles -->
+    <link rel="stylesheet" href="./styles/style.css" />
   </head>
+  <header>
+  <div class = "logotipo">
+    <img src="./images/spotify-logo.png" alt="spotify-logo"/>
+  </div>
+  
+  <div class="nav-bar"> 
+   
+     <li><a href="#">Premium</a></li>
+     <li><a href="#">Discover</a></a>   
+     <li><a href="#">Help</a></li> 
+     <li><a href="#">Download</a></li> 
+      </ul>
+    
+    </div>
+   
+  </header>
+
   <body>
```

Comment:
the body tag should come after the <head> and wrap all your content.
the <header>  is related with the content, so it should also come inside the <body>